nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
```

Comment:
Agreed, the functionality is limited to adding and removing aliases for the moment so the name should probably reflect that in some way.